instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py

```diff
@@ -48,14 +85,34 @@
 from opentelemetry.instrumentation.utils import _SUPPRESS_INSTRUMENTATION_KEY
 from opentelemetry.semconv.trace import DbSystemValues, SpanAttributes
 from opentelemetry.trace import SpanKind, get_tracer
+from opentelemetry.trace.span import Span
 from opentelemetry.trace.status import Status, StatusCode
 
+_LOG = getLogger(__name__)
+
+RequestHookT = Callable[[Span, monitoring.CommandStartedEvent], None]
+ResponseHookT = Callable[[Span, monitoring.CommandSucceededEvent], None]
+FailedHookT = Callable[[Span, monitoring.CommandFailedEvent], None]
+
+
+def dummy_callback(span, event):
```

Comment:
Why use the dummy callback and not just None as a default value?

Reply:
if I use this dummy callback, I don't need to check this hook is callable before calling it, and in general this is less error prone, we don't have the risk of invoking a None object

Reply:
Ok, I personally don't mind but perhaps the OTEL maintainers will want to keep things consistent with other instrumentations. Perhaps rename it to `noop_callback`?

Reply:
I have the same concept with the exact callback name in pika instrumentor (as suggested by their reviewer), so I believe it will be ok like that, but I don't mind to change it if they think otherwise

Reply:
Ok 👍